Unix/tools/auth_tests_setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# Used by regress and 'make tests'.  Should be run at omi/Unix level.
+
+username=$OMI_USER
+userpasswd=$OMI_PASSWORD
+
+#
+# Clean up previous (if any) running omiserver
+#
+for outputdir in output*
+do
+    pidFile=$outputdir/var/run/omiserver.pid
+    if [ -f $pidFile ]; then
+        pid=`cat $pidFile`
+
+        if [ "x${pid}" != "x" ]; then
+            sudo kill -15 $pid
```

Comment:
Seems risky. How do you know that some other process hasn't reused the pid, which can happen? This is particularly important since you're using sudo to do the kill, which allows you to kill anything on the system.

Normally, we'll do something like run `ps` looking for the pid and verifying that the process name is what you expect. That's pretty standard for this sort of thing.